ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -233,46 +233,119 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("first") {
-			pending("should start a signal then block on the first value") {
+			it("should start a signal then block on the first value") {
+				let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+
+				var result: Result<Int, NoError>?
+				dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)) {
```

Comment:
@jspahrsummers [You requested this be removed](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/1710#discussion_r24038166), but since the comment was hidden with updates I'm re-commenting on this line.

My question back:

> By pre-filling the buffer before calling `first` etc? If so, the test won't really be testing the "then block " part of the spec description.